docs/src/Scenario2/Evaluation_Scenario_2.md

```diff
@@ -49,14 +49,21 @@ ssys = structural_simplify(sys2)
 
 > Simulate for 100 days, and determine the day and level of peak total infections (sum over all the infected states I, D, A, R, T). Expected output: The peak should occur around day 47, when ~60% of the population is infected.
 
+This unit test requires deletion of the events in the SBML (or elsewhere). We deleted in the SBML.
```

Comment:
Can we elaborate on why? What's causing issues in the presence of these events? What do the events mean, etc...